test/qr_code_test.dart

```diff
@@ -29,6 +31,19 @@ void main() {
       }
     }
   });
+
+  test('fromUint8List', () {
+    for (var quality in QrErrorCorrectLevel.levels) {
+      final qr = QrCode.fromUint8List(
+          data: Uint8List.fromList([115, 104, 97, 110, 110, 97, 33]),
+          errorCorrectLevel: quality)
+        ..make();
+      for (var i = 0; i < qrModules(qr).length; i++) {
+        expect(_encodeBoolListToString(qrModules(qr)[i]),
+            qrCodeTestData[1.toString()][quality.toString()][i]);
```

Comment:
`1.toString` is a bit weird here – how about just `'1'`?

Reply:
I thought the same thing when I read line 30 of your code.